node/examples/josh_test_client.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
yeah... I felt like it was the best token to stand in for "these 4 random things I ended up with in my demo" ;-)

I guess I could sort out some weird acronym of Echo Exec Repl GrepN ;-)